src/App.jsx

```diff
@@ -1,15 +1,21 @@
-import BookList from './views/Books/BookList'
-import BookDetail from './views/Books/BookDetail'
-import './App.css'
+import { BrowserRouter as Router, Route, Switch, Link } from 'react-router-dom';
+import BookList from './views/Books/BookList';
+import BookDetail from './views/Books/BookDetail';
+import './App.css';
+import Home from './views/Home/Home';
 
 function App() {
-  // TODO: Add routes to books & views
   return (
     <main className="container">
-      <h1>Library Catalog</h1>
-      <BookList />
+      <Router>
```

Comment:
You can also add a header either here or as another component to provide links to Home and the Catalog from every page if you'd like.